app/models/org.rb

```diff
@@ -61,6 +61,7 @@ class Org < ActiveRecord::Base
   scope :managing_orgs, -> { where(abbreviation: Rails.configuration.branding[:organisation][:abbreviation]) }
   scope :funders, -> { where(org_type: 2) }
   scope :institutions, -> { where(org_type: 1) }
+  scope :non_funders, -> { where.not(org_type: 2) }
```

Comment:
yes agreed but to be honest there's no point to using a bitflag here, should have been an enum since we only allow one org_type per org. Will create a ticket to refactor after MVP

Reply:
Well, we're currently using bitflags so I think it makes sense to stick to the conventions.
Additionally, it sounds like your funder/institution organization (GIS?) is a good case for bitflags over enums as we may need combinations of the current options